packages/mountable/src/mount.tsx

```diff
@@ -16,8 +17,10 @@ const wheelBaseUrl =
 const wheelUrls = {
   stliteLib: new URL("wheels/stlite_lib-0.1.0-py3-none-any.whl", wheelBaseUrl)
     .href,
-  streamlit: new URL("wheels/streamlit-1.41.0-cp312-none-any.whl", wheelBaseUrl)
-    .href,
+  streamlit: new URL(
+    `wheels/streamlit-${STREAMLIT_VERSION}-cp312-none-any.whl`,
```

Comment:
Instead of using the imported `STREAMLIT_VERSION` value, `STREAMLIT_VERSION` should be defined as a const by `vite.config.ts`, maybe?